ocean_lib/ocean/ocean_compute.py

```diff
@@ -27,13 +29,13 @@ class OceanCompute:
 
     """Ocean assets class."""
 
-    def __init__(self, config, data_provider):
+    def __init__(self, config: Union[Config, Dict], data_provider: object) -> None:
```

Comment:
`data_provider` should be `DataServiceProvider`

Reply:
I think this can be resolved too. I fixed the ones related to DataServiceProvider. It is still weird to me that we expect a fully static class there, but this is something we need to figure out later, when removing static stuff. Definitely not now :)